src/frontend/planner_test/tests/testdata/expr.yaml

```diff
@@ -380,3 +380,17 @@
     create table t (v1 timestamp, v2 timestamp);
     select * from t where v1 >= now() or v2 >= now();
   stream_error: 'Expr error: Invalid parameter now: now expression must be placed in a comparison'
+- name: now expression condition without filter
```

Comment:
Can remove this case, it is handled in https://github.com/risingwavelabs/risingwave/pull/7167

Reply:
> Can remove this case, it is handled in #7167

fixed